src/puppetlabs/jdbc_util/core.clj

```diff
@@ -156,3 +156,31 @@
                        [sql 0]
                        seq-params-w-indices)]
     (vec (conj (flatten parameters) sql'))))
+
+(defn has-extension [db extension]
+  (-> (jdbc/query db ["select count(*) from pg_extension where extname = ?" extension])
+      first
+      :count
+      pos?))
+
+(defn has-pglogical-extension [db]
+  (has-extension db "pglogical"))
+
+(defn- unsafe-escape-sql-string
```

Comment:
fwiw, this is a blatant imitation of https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringEscapeUtils.html#escapeSql(java.lang.String) (which has since been deprecated, as it's a horrible idea). But yeah, I can make the name more descriptive.